<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="delta_parameters">

<xacro:property name="base_radius" value="0.3"/>
<xacro:property name="base_depth" value="0.02"/>

<xacro:property name="upper_length" value="0.3"/>
<xacro:property name="upper_radius" value="0.01"/>

<xacro:property name="lower_radius" value="0.01"/>
<xacro:property name="lower_horizontal_length" value="0.4"/>
<xacro:property name="lower_vertical_length" value="0.6"/>

<xacro:property name="joint_lower_limit" value="${-pi/2}"/>
<xacro:property name="joint_upper_limit" value="${pi/2}"/>

<xacro:property name="end_effector_radius" value="0.2"/>
<xacro:property name="end_effector_depth" value="0.03"/>

<xacro:property name="base_mass" value="0.1"/>
<xacro:property name="upper_mass" value="0.04"/>
<xacro:property name="lower_horizontal_mass" value="0.02"/>
<xacro:property name="lower_vertical_mass" value="0.04"/>
<xacro:property name="end_effector_mass" value="0.05"/>

<xacro:property name="break_indicator_radius" value="0.03"/>

</robot>
